image/templates/helm/shared/config-templates/scanner-v4/matcher-config.yaml.tpl

```
@@ -29,7 +29,7 @@ matcher:
 {{- end }}
       client_encoding=UTF8
     password_file: /run/secrets/stackrox.io/secrets/password
-  vulnerabilities_url: https://central.{{ .Release.Namespace }}.svc/api/extensions/scannerdefinitions?version=ROX_VULNERABILITY_VERSION
+  vulnerabilities_url: https://storage.googleapis.com/scanner-v4-test/vulnerability-bundles/yli3-enrichEPSS/vulnerabilities.zip
```

don't forget to revert this change prior to merge 😄 

```suggestion
  vulnerabilities_url: https://central.{{ .Release.Namespace }}.svc/api/extensions/scannerdefinitions?version=ROX_VULNERABILITY_VERSION
```